<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1580</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Оценка тональности текста</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>1461</width>
      <height>871</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_single">
     <attribute name="title">
      <string>Одиночный анализ</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>Введите текст для оценки тональности:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>100</red>
              <green>102</green>
              <blue>240</blue>
             </color>
            </brush>
           </colorrole>
          </active>
         </palette>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Оценить тональность текста</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Результат оценки: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_file">
     <attribute name="title">
      <string>Анализ файла</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выберите CSV файл с отзывами:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit_filepath">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Путь к файлу CSV</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_browse">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Обзор...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_column">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Столбец с отзывами:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_column">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>Имя или индекс столбца</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_delimiter">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Разделитель:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_delimiter">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>Запятая (,)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Точка с запятой (;)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Табуляция (Tab)</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_analyze">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Анализировать файл</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Результаты анализа</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_total">
           <property name="text">
            <string>Всего отзывов: 0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_star1">
           <property name="text">
            <string>1 звезда: 0 (0%)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_star2">
           <property name="text">
            <string>2 звезды: 0 (0%)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_star3">
           <property name="text">
            <string>3 звезды: 0 (0%)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_star4">
           <property name="text">
            <string>4 звезды: 0 (0%)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_star5">
           <property name="text">
            <string>5 звезд: 0 (0%)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_results">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Здесь будет отображаться подробная информация о результатах</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1580</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>